rtm.go

```diff
@@ -64,12 +59,7 @@ func (api *Client) ConnectRTMContext(ctx context.Context) (info *Info, websocket
 	// Fixed by: https://github.com/golang/net/commit/5058c78c3627b31e484a81463acd51c7cecc06f3
 	// but slack returns the address with no port, so we have to fix it
```

Comment:
@apokalyptik can you please update this comment and remove the `websocketizeURLPort()` function?